src/estop.cpp

```diff
@@ -34,20 +35,22 @@ bool toggleEstopDisable(std_srvs::Trigger::Request &req, std_srvs::Trigger::Resp
 */
 void mySigintHandler(int sig)
 {
-  gpioSetValue(ESTOP_PIN, 0);
-  ros::shutdown();
+    gpioSetValue(ESTOP_PIN, 0);
+    ros::shutdown();
 }
 
 /**
  * Initialize the configuration to control the GPIO pin.
  */
-void initializeGPIO() {
+void initializeGPIO()
+{
     gpioExport(ESTOP_PIN);
     gpioSetDirection(ESTOP_PIN, 1);
     gpioSetValue(ESTOP_PIN, 1);
 }
 
-int main(int argc, char **argv) {
+int main(int argc, char **argv)
```

Comment:
Also, was this tested in any way ? 

Looking up how to disable the e-stop once we want to shut down the node seems to require a little more code :

https://answers.ros.org/question/27655/what-is-the-correct-way-to-do-stuff-before-a-node-is-shutdown/